tests/lean/interactive/completion2.lean.expected.out

```diff
@@ -1,31 +1,53 @@
 {"textDocument": {"uri": "file://completion2.lean"},
  "position": {"line": 19, "character": 10}}
 {"items":
- [{"label": "ex2", "detail": "?a ≤ ?b → ?a + 2 ≤ ?b + 2"},
-  {"label": "ex3", "detail": "?a ≤ ?b → ?c ≤ ?d → ?a + ?c ≤ ?b + ?d"},
-  {"label": "ax1", "detail": "?a ≤ ?b → ?a - ?a ≤ ?b - ?b"},
-  {"label": "ex1", "detail": "?a ≤ ?b → ?a + ?a ≤ ?b + ?b"}],
+ [{"label": "ex2", "detail": "LE.le (α := Nat) ?a ?b → ?a + 2 ≤ ?b + 2"},
```

Comment:
These regressions look quite bad :(

Reply:
Fixed in https://github.com/leanprover/lean4/commit/50e6267e74af3f47e4cd251976a939f7b1498255